tails_files/install.sh

```diff
@@ -277,14 +277,6 @@ EOL
   fi
 }
 
-function configure_ansible_apt_persistence()
-{
-  # set ansible to auto-install
-  if ! grep -q 'ansible' "${tails_live_persistence}/live-additional-software.conf"; then
-    echo "ansible" >> "${tails_live_persistence}/live-additional-software.conf"
```

Comment:
Currently running instances installed after 0.3.7 will have "ansible" in the apt persistence config, it's not enough simply to remove the setup logic from the install.sh. The right solution seems to be to disable apt persistence for Ansible by snipping out the line from persistence.conf if it exists, but the Tails 2 -> 3 transition here saves us, since Admins will need to configure new sticks from scratch anyway.